test/test_utils.py

```diff
@@ -994,6 +994,12 @@ def test_js_to_json_edgecases(self):
         on = js_to_json('{42:4.2e1}')
         self.assertEqual(json.loads(on), {'42': 42.0})
 
+        on = js_to_json('{ "0x40": true }')
```

Comment:
`true` is irrelevant here, keep tests minimal.

Reply:
Do you have a preferred value? I don't think `{ "0x40": }` would be a meaningful test case since it's neither valid JS nor JSON.

Reply:
You can either use an array `["0x40"]` or a dict with both key and value set to `"0x40"`.